<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopoConstraints</class>
 <widget class="QWidget" name="TopoConstraints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Topological constraints</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="l_label_animations">
          <property name="text">
           <string>Enable animations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_animations">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="l_label_constraints">
          <property name="text">
           <string>No. constraints:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_constraints">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tw_constraints_list">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Component 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Component 2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>